include/worker.hpp

```diff
@@ -100,6 +100,16 @@ class Worker
                       types::ObjectMap& objectInterfaceMap,
                       const std::string& vpdFilePath);
 
+    /**
+     * @brief API to get VPD config JSON object
+     *
+     * @return parsed VPD config JSON object
+     */
+    inline nlohmann::json getConfig() const
```

Comment:
> getConfig

Be more specific, getParsedSystemConfig or something like that.

Reply:
done